Automattic/crowdsignal-ui
# Update Isolated Block Editor

This patch updates the Automattic/isolated-block-editor to `2.9.0`, bumps up versions for our tooling and fixes any issues that arose from that - mostly changes in the linter.  

It's worth noting the updated version of the block editor fixes the issue with block previews (examples, block styles etc.) - they should now appear correctly. Albeit still without proper styling.  

# Testing

- Run `yarn install` and `yarn start`
- Everything should build correctly
- Verify our new editor still works, there shouldn't really be any visible difference except for the previews.
